### PR TITLE
Document config files

### LINKED ISSUE #103: Best approach to edit PG conf files?
What's the right approach to edit the postgres server *.conf files?
Eg, if I want to change the work_mem or connection limits.

### PR DESCRIPTION
**Pull request by joshco**
fixes #103

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,3 +239,18 @@ dokku postgres:backup-schedule lolipop CRON_SCHEDULE BUCKET_NAME
 # remove the scheduled backup from cron
 dokku postgres:backup-unschedule lolipop
 ```
+
+## Postgres Configuration
+
+If you wish to tune the postgres instances various .conf files, you can find them by using the postgres:info command.
+
+> BEWARE: The earlier documentation tries to mislead you.  You will find the config files in the data directory __not__ the config directory.
```
Comment (josegonzalez) [inline README.md]: Why not either update the docs or fix the underlying bug?

Comment (joshco) [inline README.md]: Because at the time I was more interested in making a wise crack.
I'll take a look now

**Comment by joshco** [comment]
updated.  -- this is all I have time for at the moment.

**Comment by rvalyi** [comment]
I approve this, but I think this line should also be deleted now:
https://github.com/dokku/dokku-postgres/blob/master/functions#L24

I was investigating how to customize the configuration and the presence of this config-dir thing is indeed confusing, making you search at the wrong place. So I think this should be completed and merged.

**Pull request merged**